src/components/status/utils/varselHooks.ts

```diff
@@ -0,0 +1,21 @@
+import { UseMutationResult } from "@tanstack/react-query";
+import { useEffect } from "react";
+import { Gyldighetstidspunkt } from "../../../types/oppfolgingsplan";
+
+export const useFerdigstillVarsel = (
+  ferdigstillVarsel: UseMutationResult<void, unknown, number, unknown>,
+  oppfolgingsplanId: number,
+  gyldighetstidspunkt: Gyldighetstidspunkt | undefined
+) => {
+  useEffect(() => {
+    if (oppfolgingsplanId && gyldighetstidspunkt) {
```

Comment:
Hva er grunnen til å sjekke gyldighetstidspunkt?

Reply:
Kanskje unødvendig. Ser det gjøres for at planene vises, men kan ta det bort 👍

Reply:
@kaja-nav Fikset 👍